test/system/system_suite_test.go

```diff
@@ -29,17 +28,63 @@ func TestSystem(t *testing.T) {
 
 var _ = SynchronizedBeforeSuite(func() {
 	//this runs once for the whole suite
+	worker = &destination{
+		context:       getEnvOrDefault("WORKER_CONTEXT", "kind-worker"),
+		checkExitCode: true,
```

Comment:
nit: this seems to be the only value we care about? maybe `ignoreExitCode` which defaults to `false` would be better?

Reply:
sounds good! I'll refactor